riot/riot.py

```diff
@@ -345,53 +342,44 @@ def pythonpath(self) -> str:
 
         return ":".join(paths)
 
+    def match_venv_pattern(self, pattern):
```

Comment:
docstring, maybe with some examples, might be nice

Reply:
tests?

Reply:
This is tested with `test_list_with_venv_pattern` and `test_run_venv_pattern`, but I'm happy to add some unit tests if you think that's worth. What makes me a bit uneasy is that `venv_pattern` is no longer the right wording as we're not matching a venv anymore. Changing this would break the CLI though.

Reply:
hmm, right

this is where maybe some unit tests would be good, we can enumerate all the possible/expected matching/patterns we support

Reply:
(or we can expand the others tests to cover them too)